src/DiffSharp.Backends.Torch/Torch.RawTensor.fs

```diff
@@ -73,7 +74,7 @@ type TorchRawTensor(tt: TorchTensor, shape: int[], dtype, device) =
             let stop = fullBounds.[i,1] + 1
 
             let len = stop - start
-            let idxs = LongTensor.Arange(int64 start, int64 stop, 1L, device=toTorchDevice t.Device)
+            use idxs = LongTensor.Arange(int64 start, int64 stop, 1L, device=toTorchDevice t.Device)
```

Comment:
This line is not essential.  But it indicates how we can get deterministic disposal of tensors when we need to (the `Arange` tensor and its backing unmanaged resources are disposed as soon as `idxs` is no longer in scope).  This is likely to become important for performance as time goes on.

Reply:
Thank you, this is a good example.